### PR TITLE
Add contact info to CUB - Cube Post -Update facilities.json

### PR DESCRIPTION
**Pull request by japierce1028**
Opt IN

### CODE REVIEW

**Comment by japierce1028** [comment]
The contact info was for "Cube Post" and the replacement was "Cubic Post". I'll confirm it's the same facility.

**Comment by japierce1028** [comment]
Changed to CPB

Begin forwarded message:
From: Nacho Melero <nacho.melero@cube-post.com>
Subject: Re: Digital Cinema Naming - Request for CUB
Date: October 11, 2024 at 8:55:52 AM MDT
To: Jerry Pierce <jerry@jerrypierce.org>
Cc: digitalcinemanaming@isdcf.com

Hi Jerry.

You're right, we're not representing Cubic Post. Please, could you assign CPB as a code? It means "Cube Post Barcelona"

Thank you,



Nacho Melero
+34 660298436  IMDb
www.cube-post.com

File: src/main/data/facilities.json [hunk 1]
```diff
@@ -1523,7 +1533,7 @@
   },
   {
     "code": "CUB",
-    "description": "Cubic Post"
+    "description": "Cube Post"
```
Comment (palemieux) [inline src/main/data/facilities.json:1536]: @japierce1028 Need to revert this change

**Review by palemieux — APPROVED** [review]
